server/models/Profile.js

```diff
@@ -0,0 +1,51 @@
+const mongoose = require("mongoose");
+
+const profileSchema = new mongoose.Schema({
+  email: {
```

Comment:
Please remove the `email` field as it already exists in the User document

Reply:
I use email as the key for profile

Reply:
@JunZheng007 Mongoose creates unique id's for each document which we can use to identify them